Mathlib/Analysis/NormedSpace/OperatorNorm.lean

```diff
@@ -464,22 +453,30 @@ theorem op_norm_comp_le (f : E →SL[σ₁₂] F) : ‖h.comp f‖ ≤ ‖h‖ *
       exact h.le_op_norm_of_le (f.le_op_norm x)⟩
 #align continuous_linear_map.op_norm_comp_le ContinuousLinearMap.op_norm_comp_le
 
+-- Porting note: restatement of `op_norm_comp_le` for linear maps.
+/-- The operator norm is submultiplicative. -/
+theorem op_norm_comp_le' (h : Eₗ →L[𝕜] Fₗ) (f : E →L[𝕜] Eₗ) : ‖h.comp f‖ ≤ ‖h‖ * ‖f‖ :=
+  op_norm_comp_le h f
+
 theorem op_nnnorm_comp_le [RingHomIsometric σ₁₃] (f : E →SL[σ₁₂] F) : ‖h.comp f‖₊ ≤ ‖h‖₊ * ‖f‖₊ :=
   op_norm_comp_le h f
 #align continuous_linear_map.op_nnnorm_comp_le ContinuousLinearMap.op_nnnorm_comp_le
 
-omit σ₁₃
-
 /-- Continuous linear maps form a seminormed ring with respect to the operator norm. -/
 instance toSemiNormedRing : SeminormedRing (E →L[𝕜] E) :=
   { ContinuousLinearMap.toSeminormedAddCommGroup, ContinuousLinearMap.ring with
     norm_mul := fun f g => op_norm_comp_le f g }
 #align continuous_linear_map.to_semi_normed_ring ContinuousLinearMap.toSemiNormedRing
 
+-- Porting FIXME: replacing `(algebra : Algebra 𝕜 (E →L[𝕜] E))` with
+-- just `algebra` below causes a massive timeout.
```

Comment:
Is this still true?

Reply:
It's either this or `set_option maxHeartbeats 2000000` (2 million). 1 million is not enough.

Reply:
Thanks for confirming!